### ISSUE TITLE
Problem with TBitmap.ToSkImage Helper

### ISSUE THREAD

**Issue by fschenckel** [skia4delphi/skia4delphi#252]
With the Beta3 version, the following code generates an AV with 0x0000 address when FSKImage is freed. It works as expected with the Beta1.
(Note: I know the code is useless, it's just to show the problem)

```
  var LBitmap := TBitmap.Create;
  try
    LBitmap.LoadFromFile('.\Image1.bmp');

    FSKImage := LBitmap.ToSkImage;
    FSKImage := nil;
  finally
    LBitmap.Free;
  end;

```

**Comment by viniciusfbb** [comment]
Fixed on commit 792d6965e2f77cc260ccf2c60483b698253c71f6
It will be available on next release

**Comment by viniciusfbb** [comment]
Fixed on new version [v6.0.0-beta4](https://github.com/skia4delphi/skia4delphi/releases)
